apify_handler.py

```
@@ -1,349 +1,365 @@
+            logger.warning(f"No tweet data found for URL: {url}")
+            return None
+
+        # Get the first (and should be only) item
+        tweet_data = dataset_items[0]
+        
+        # Log success
+        logger.info(f"Successfully fetched tweet from URL: {url}")
+        
+        return tweet_data
+
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)